clients/core/src/publicPages/application/pages/ApplicationAuthenticated/ApplicationAuthenticated.tsx

```diff
@@ -50,8 +51,9 @@ export const ApplicationAuthenticated = (): JSX.Element => {
     mutationFn: (modifiedApplication: PostApplication) => {
       return postNewApplicationAuthenticated(phaseId ?? 'undefined', modifiedApplication)
     },
-    onSuccess: () => {
+    onSuccess: (data) => {
       queryClient.invalidateQueries({ queryKey: ['application', phaseId, user?.email] })
+      setConfirmationMailSent(data.confirmationMailSent)
```

Comment:
If it's necessary, then it would be the same here.


```suggestion
      setConfirmationMailSent(data.confirmationMailSent ?? false)
```